frontend/packages/dev-console/locales/en/devconsole.json

```diff
@@ -434,8 +434,8 @@
   "Rate of transmitted packets dropped": "Rate of transmitted packets dropped",
   "Average Container bandwidth by Pod: received": "Average Container bandwidth by Pod: received",
   "Average Container bandwidth by Pod: transmitted": "Average Container bandwidth by Pod: transmitted",
-  "Project access allows you to add or remove a user&apos;s access to the project. More advanced management of role-based access control appear in <2>Roles</2> and <5>Role Bindings</5>. For more information, see the <8>role-based access control documentation</8> .": "Project access allows you to add or remove a user&apos;s access to the project. More advanced management of role-based access control appear in <2>Roles</2> and <5>Role Bindings</5>. For more information, see the <8>role-based access control documentation</8> .",
```

Comment:
Are this change requested by someone or should we keep the old version? Also if we don't have a translation for this strings now, we should avoid unnecessary changes. If there is a reason for that I'm absolutely fine with that change!